instrumentation/github.com/gin-gonic/gin/otelgin/gintrace_test.go

```diff
@@ -10,12 +10,17 @@ import (
 	"net/http"
 	"net/http/httptest"
 	"testing"
+	"time"
 
 	"github.com/gin-gonic/gin"
 	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
 
 	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/attribute"
 	"go.opentelemetry.io/otel/propagation"
+	sdktrace "go.opentelemetry.io/otel/sdk/trace"
```

Comment:
Sorry I missed this earlier. The instrumentations should not rely on the sdk, only the API. So we should either update this test to not rely on the SDK, or move it to a `test` subfolder.

Reply:
Done